test/fixtures/full_config.yml

```diff
@@ -3522,6 +3524,11 @@ Style/RedundantConditional:
   Description: Don't return true/false from a conditional.
   Enabled: false
   VersionAdded: '0.50'
+Style/RedundantEach:
+  Description: Checks for redundant `each`.
+  Enabled: pending
+  Safe: false
+  VersionAdded: '1.38'
```

Comment:
@rafaelfranca shall we enable this? It seems useful, although my concern would be false positives. We encountered one in one of our repos, but it and others seem to have been fixed in 1.39.